### DISCUSSION TITLE
Chia and eterhum mimngin Lhr problem

### DISCUSSION THREAD

**Discussion by sencer1453** [Chia-Network/chia-blockchain#11182] [Support]
I've been digging for chia and ethereum for a few months now. On the same computer, I open lolminer or trex first. After a few minutes, I turn on the device. Then Lhr detection enters and Etherium mining stops. Please help me.

**Comment by loppefaaret** [comment]
Do you have any problems with farming Chia ? if you problem is with lolminer, perhaps try and see what you can find there ?
